galaxy_ng/app/api/ui/views/auth.py

```diff
@@ -44,6 +45,10 @@ def get(self, request, *args, **kwargs):
 
     def post(self, request, *args, **kwargs):
         serializer = self.get_serializer(data=self.request.data)
+
+        if str(request.user) == "AnonymousUser" and settings.GALAXY_ENABLE_VIEW_ONLY_ACCESS:
```

Comment:
What is this in here for? It seems like this should break authentication when running in view only mode.

Reply:
Doh!  Had that in there when testing things, forgot to remove it.  Thanks!